stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -347,12 +365,17 @@ public DefaultInspectorModulesBuilder remove(String moduleName) {
               new RhinoDetectingRuntimeReplFactory(mContext)));
       provideIfDesired(new Worker());
       if (Build.VERSION.SDK_INT >= DatabaseConstants.MIN_API_LEVEL) {
-        provideIfDesired(
-            new Database(
-                mContext,
-                mDatabaseFiles != null ?
-                    mDatabaseFiles :
-                    new DefaultDatabaseFilesProvider(mContext)));
+        Database database = new Database();
+        database.add(new SqliteDatabaseDriver(mContext,
+            mDatabaseFilesProvider != null ?
+                mDatabaseFilesProvider :
+                new DefaultDatabaseFilesProvider(mContext)));
+        if (mDatabaseDrivers != null) {
+          for (Database.DatabaseDriver databaseDriver : mDatabaseDrivers) {
```

Comment:
nit: this syntax evaluates to a slow expression on Dalvik devices.  I usually get in the habit of writing:

```java
for (int i = 0, N = mDatabaseDrivers.size(); i < N; i++) {
  database.add(mDatabaseDrivers.get(i));
}
```

It's almost the same amount of typing but doesn't allocate the iterator and the pattern also works to iterate things like `SimpleArrayMap` and `SparseArray`.